devel/libecl/src/ecl_grid.c

```diff
@@ -5790,6 +5850,48 @@ void ecl_grid_reset_actnum( ecl_grid_type * grid , const int * actnum ) {
 }
 
 
+static void  ecl_grid_fwrite_self_nnc( const ecl_grid_type * grid , fortio_type * fortio ) {
+  const int default_index = 1;
+  int_vector_type * g1 = int_vector_alloc(0 , default_index );
+  int_vector_type * g2 = int_vector_alloc(0 , default_index );
+  int g;
+
+  for (g=0; g < ecl_grid_get_global_size(grid); g++) {
+    ecl_cell_type * cell = ecl_grid_get_cell( grid , g );
+    const nnc_info_type * nnc_info = cell->nnc_info;
+    if (nnc_info) {
+      const nnc_vector_type * nnc_vector = nnc_info_get_self_vector(nnc_info);
+      int i;
+      for (i = 0; i < nnc_vector_get_size( nnc_vector ); i++) {
+        int nnc_index = nnc_vector_iget_nnc_index( nnc_vector , i );
+        int_vector_iset( g1 , nnc_index , 1 + g );
```

Comment:
Just to be sure; the NNC in the grid is "directed", as in the sense that it will only be stored in one of the cells?  Otherwise, won't we here write each NNC twice?